backend/api/Services/MapService.cs

```diff
@@ -173,14 +173,14 @@ private int[] ExtractImageSize(BlobItem map)
                 int y = int.Parse(map.Metadata["imageHeight"], CultureInfo.CurrentCulture);
                 return new int[] { x, y };
             }
-            catch (FormatException e)
+            catch (Exception e) when (e is FormatException || e is KeyNotFoundException)
```

Comment:
We should probably catch all errors here to log the warning and reraise
```suggestion
            catch (Exception e)
```